File: Practice/SampleFragment2/app/src/main/java/org/techtown/samplefragment/MainActivity.java
package org.techtown.samplefragment;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.FragmentManager;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity implements ListFragment.ImageSelectionCallback {

    ListFragment listFragment;
    ViewerFragment viewerFragment;

    int[] images = {R.drawable.dream01, R.drawable.dream02, R.drawable.dream03};

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        FragmentManager manger= getSupportFragmentManager();
        listFragment = (ListFragment) manger.findFragmentById(R.id.listFragment);
        viewerFragment = (ViewerFragment) manger.findFragmentById(R.id.viewerFragment);
    }

    public void onImageSelected(int position) {
        viewerFragment.setImage(images[position]);
    }
}

File: Practice/Dagger2/app/src/main/java/org/techtown/dagger2/MainActivity.kt
package org.techtown.dagger2

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import org.jetbrains.annotations.TestOnly

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

    }

    public class ExampleUnitTest {
        @TestOnly
        public testHelloWorld() {
            MyComponent myComponent = DaggerMyComponent.create()

        }
    }
}